core/src/scss/elements/input/_radio-checkbox.scss

```diff
@@ -0,0 +1,85 @@
+@charset "UTF-8";
+
+// Custom checkboxes and radio buttons.
+[type='checkbox'],
+[type='radio'] {
+  // The actual input element is only visible to screen readers, because
+  // all visual styling is done via the label.
+  @include hide-visually;
+
+  .lt-ie9 & {
```

Comment:
Where does this class come from?

Reply:
@sherakama , I actually didn't do any work to "improve" some of the current input styles nor look at them that closely since the purpose of the PR is just to split the input.scss file into smaller chunks so developers can cherry pick only the parts they need for their projects. I agree that many things need work (quite a bit in fact) for our current Decanter input styles, but I'd like to save that for another PR. Let me know what you think.